bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcComponentParamsMapper.kt

```diff
@@ -29,6 +30,7 @@ internal class BcmcComponentParamsMapper(
             clientKey = clientKey,
             isAnalyticsEnabled = isAnalyticsEnabled ?: true,
             isCreatedByDropIn = false,
+            amount = Amount.EMPTY,
```

Comment:
Should we add amount to configurations in this PR?

Reply:
Let's have separate PRs to not block the branch.